service/service-samples/sample-service-reactive-redis-book/src/main/resources/service-book-application.properties

```diff
@@ -0,0 +1,18 @@
+#/*
```

Comment:
remove the jadadoc stype comment and only use #

Reply:
done